connectors/src/connectors/slack/temporal/client.ts

```diff
@@ -55,17 +59,22 @@ export async function launchSlackSyncOneThreadWorkflow(
 
   const workflowId = `slackSyncOneThreadWorkflow-${connectorId}-${threadTs}`;
   try {
-    const handle = await client.workflow.start(syncOneThread, {
-      args: [
-        connectorId,
-        dataSourceConfig,
-        nangoConnectionId,
-        channelId,
-        threadTs,
-      ],
-      taskQueue: "slack-queue",
-      workflowId: workflowId,
-    });
+    const handle = await client.workflow.signalWithStart(
```

Comment:
`signalWithStart()` will make sure a new workflow is started or an existing one will get the signal.